scss/components/_radio.scss

```diff
@@ -17,13 +17,16 @@
 
           @media screen and (-ms-high-contrast: active) {
             border-color: Highlight;
+            border-image: url("data:image/svg+xml;charset=utf8,<svg xmlns='http://www.w3.org/2000/svg'><rect width='100%' height='100%' rx='10' fill='Highlight'/></svg>");
```

Comment:
@geppy Adding `border-image` using the `rect` is not showing in the circular shape.Attaching screenshot for your reference.
![image](https://user-images.githubusercontent.com/58989705/86354673-12c92380-bc87-11ea-9fff-61c8f7f4c50c.png)

Reply:
What is the computed value for `border-radius`?

Reply:
I checked the computed value of border-radius its 50% . Attaching screenshot for your reference 
![radioBorder](https://user-images.githubusercontent.com/58989705/86474937-6f990c80-bd61-11ea-8ea7-4adce75a6af6.JPG)

Reply:
The "Slim" radio button has the correct border radius.  What's different there?

Why did you specify `rx`?

Reply:
**The "Slim" radio button has the correct border-radius. What's different there?**
`border-radius` is not working in the `border-image` but it's working on the border with the solid color but I can't able to change the color as a highlight in the high contrast mode.

Why did you specify rx?
I specified rx to make the `rect` as rounded corners to bring it as circular one.

Reply:
I tried these to make the border-image circular

1. Using Circle SVG with Stroke - `Not working`
![image](https://user-images.githubusercontent.com/58989705/86780072-f4459c80-c079-11ea-82d9-62752efa4cf2.png)

2. Using Rect SVG with Stroke and Border image slice - `Not Working`
![image](https://user-images.githubusercontent.com/58989705/86780462-67e7a980-c07a-11ea-816d-a1616d2a6a36.png)

3. Tried using the Background image instead of border-image with Circle SVG and making the border as none to remove the existing border-style: - `Working`
![image](https://user-images.githubusercontent.com/58989705/86786588-cf552780-c081-11ea-976d-fb3103265b27.png)
Here is the code for your reference
`.radio input[type="radio"]:checked + label .radio-icon::after {
    background-image: url("data:image/svg+xml;charset=utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='4' fill='highlight' stroke='window' stroke-width='2' /></svg>");
  }

  .radio input[type="radio"]:checked + label .radio-icon {
    background-image: url("data:image/svg+xml;charset=utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='4' stroke='highlight' stroke-width='1.5' /></svg>");
    border: none;
  }`